GitUI/UserControls/RevisionGrid/RevisionGridControl.Designer.cs

```diff
@@ -232,7 +235,6 @@ private void InitializeComponent()
             // 
             this.createNewBranchToolStripMenuItem.Image = global::GitUI.Properties.Images.BranchCreate;
             this.createNewBranchToolStripMenuItem.Name = "createNewBranchToolStripMenuItem";
-            this.createNewBranchToolStripMenuItem.ShortcutKeys = ((System.Windows.Forms.Keys)((System.Windows.Forms.Keys.Control | System.Windows.Forms.Keys.B)));
```

Comment:
Note: Not in this PR, the shortcut keys removed in this file are not in Settings

Reply:
This shortcut still works without issue from the menu-bar hotkey, which is reassignable via the settings page.